galaxy_ng/app/access_control/statements/insights.py

```diff
@@ -39,7 +39,7 @@
             "principal": "authenticated",
             "effect": "allow",
             "condition": [
-                "has_model_perms:ansible.delete_ansible_repo_content",
+                "has_model_perms:ansible.delete_collection",
```

Comment:
@rochacbruno I renamed this to match the permission already in the database via pulp_ansible https://github.com/pulp/pulp_ansible/pull/342#discussion_r469434790